src/main/webapp/resources/js/components/ncbi/export-table/NcbiExportTable.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Table } from "antd";
+import {
+  getProjectNCBIExports,
+  NcbiExportSubmissionTableModel,
+} from "../../../apis/export/ncbi";
+import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
+import { setBaseUrl } from "../../../utilities/url-utilities";
+import { getPaginationOptions } from "../../../utilities/antdesign-table-utilities";
+import { Link, useLoaderData } from "react-router-dom";
+import NcbiUploadStateTag from "../ExportUploadStateTag/NcbiUploadStateTag";
+import { UserMinimal } from "../../../types/irida";
+import { ExportUploadState } from "../../../types/irida/ExportUpoadState";
+import { DataFunctionArgs } from "@remix-run/router/utils";
+
+/**
+ * React router data loader (https://beta.reactrouter.com/en/dev/route/loader)
+ * Fetches a list of all NCBI SRA Exported submissions for this project.
+ * @param params
+ */
+export async function loader({
+  params,
+}: DataFunctionArgs): Promise<NcbiExportSubmissionTableModel[]> {
+  if (params.projectId) {
+    return getProjectNCBIExports(parseInt(params.projectId));
+  } else {
+    return Promise.reject("Requires a project id");
```

Comment:
Internationalize

Reply:
Updated in 4b18e42

Reply:
Looks like that is a different commit. This one still needs to be internationalized

Reply:
Fine try this one :) 009a7f2

Reply:
Much better! :) Thanks!